lib/auth/join_bound_keypair.go

```diff
@@ -643,6 +643,9 @@ func (a *Server) RegisterUsingBoundKeypairMethod(
 		nil, // TODO: extended claims for this type?
 		nil, // TODO: workload id claims
 	)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
```

Comment:
Mildly disturbed that this missing error check didn't get caught by the linter 👀 

Mostly harmless, I think, since this:
1. was previously more or less infallible since the generation counter wasn't being checked in the first place
2. resulted in returning an empty cert bundle, so clients would just falsely think they were given credentials

It could've locked clients out unintentionally by e.g. consuming a registration secret and not returning certs, but that's about it.